model/riscv_pmp_control.sail

```diff
@@ -92,6 +92,21 @@ function pmpAddrRange(cfg: Pmpcfg_ent, pmpaddr: xlenbits, prev_pmpaddr: xlenbits
   }
 }
 
+/* Check mseccfg.MMl=1 and mseccfg.MMWP=0 and make execute false for Mmode for matching region not found*/
+
+val MMLcheck: (AccessType(ext_access_type)) -> bool effect {rreg}
+function MMLcheck(acc) = {
+  if ( mseccfg.MML() == 0b1)
+	then { 
+		    match acc {
+				Execute(_)  => false,
+      		   _       => true
+			  }
+			 }
+	else
+	     true	 
```

Comment:
Formatting is unreadable, let alone conforming

Reply:
Formatting is fixed, readability is improved